src/main/cpp/RLBotInterface/RLBotInterface/GameFunctions/GameFunctions.hpp

```diff
@@ -15,7 +15,7 @@ extern "C"
 	namespace GameFunctions
 	{
 		DLL_EXPORT RLBotCoreStatus RLBOT_CORE_API UpdateLiveDataPacket(LiveDataPacket* pLiveData);
-		DLL_EXPORT RLBotCoreStatus RLBOT_CORE_API UpdateMatchDataPacket(MatchDataPacket* pMatchData);
+		// DLL_EXPORT RLBotCoreStatus RLBOT_CORE_API UpdateMatchDataPacket(MatchDataPacket* pMatchData);
```

Comment:
Because it was never functional, and commenting it out allowed me to get rid of the MatchDataPacket struct. My intent is to use boost queues for setting game state.

Reply:
so this is a temporary commented out function?

What method will external clients be calling to update game state?

Reply:
okay sg